util/kubeconfig/kubeconfig.go

```diff
@@ -2,6 +2,7 @@ package kubeconfig
 
 import (
 	"encoding/base64"
+	goErrors "errors"
```

Comment:
Is there a conflict or did your editor automatically do this before the errors package was removed?